code/optimize/src/main/java/com/adobe/marketing/mobile/optimize/AEPOptimizeError.kt

```diff
@@ -44,16 +105,18 @@ class AEPOptimizeError(val type: String? = "", val status: Int? = 0, val title:
                 AdobeError.UNEXPECTED_ERROR
             )
         }
+
+        private fun getAdobeErrorFromStatus(status: Int?): AdobeError = when {
```

Comment:
@ishita-gambhir-adobe and @akhiljain1907 in case of status is null, we're still getting the `UNEXPECTED_ERROR` is this expected or should it return null in that case since you've mentioned adobeError nullable above